app/models/procedure_revision_type_de_champ.rb

```diff
@@ -46,6 +46,10 @@ def upper_coordinates
       upper += parent.upper_coordinates
     end
 
+    if type_de_champ.private?
+      upper += revision.revision_types_de_champ_public.filter { !_1.block? && !_1.child? }
```

Comment:
```suggestion
      upper += revision.revision_types_de_champ_public
```

car `types_de_champ_public` ne comporte pas d'enfant et que le composant condition filtre les types de champs et rejette les block

Reply:
Bien vu ! 👍